src/assets/stylesheets/shared/main.scss

```diff
@@ -49,6 +49,10 @@ body {
   position: relative;
 }
 
+.none-assigned {
```

Comment:
is the unassigned used anywhere other than the detail panel? can it just be kept to that component?